python/prairielearn.py

```diff
@@ -449,14 +452,20 @@ def from_json(v):
     return v
 
 
-def inner_html(element):
-    inner = element.text
-    if inner is None:
-        inner = ""
-    inner = html.escape(str(inner))
-    for child in element:
-        inner += lxml.html.tostring(child, method="html").decode("utf-8")
-    return inner
+def inner_html(element: lxml.html.HtmlElement) -> str:
```

Comment:
Benchmarks of new vs old:

![image](https://user-images.githubusercontent.com/1345068/222844822-53bba0d4-8125-439b-96e3-18246b4219c0.png)

Not a huge difference but essentially free performance for rewriting a very short function.